image_tools/CMakeLists.txt

```diff
@@ -45,33 +45,18 @@ if(BUILD_TESTING)
   find_package(ament_lint_auto REQUIRED)
   ament_lint_auto_find_test_dependencies()
 
+  # get the rmw implementations ahead of time
   find_package(rmw_implementation_cmake REQUIRED)
+  get_available_rmw_implementations(rmw_implementations)
+  foreach(rmw_implementation ${rmw_implementations})
+    find_package("${rmw_implementation}" REQUIRED)
+  endforeach()
```

Comment:
The reason that I thought it was necessary is because it seems from [your comment here](https://github.com/ros2/ros2/issues/302#issuecomment-276519274) that these lines [were not removed from the tests mentioned](https://github.com/ros2/system_tests/blob/3005b6df5261098bc6a51567b57f1022f7ef8830/test_rclcpp/CMakeLists.txt#L33..L38) for a reason that also applies here (we are making test executables for each rmw implementation).

Will the `call_for_each_rmw_implementation` macro work fine without doing `find_package` on each of the rmw implementations? I thought it must be necessary but since [that macro calls `get_all_rmw_implementations`](https://github.com/ros2/rmw/blob/847784e6a0b714630c12a67eb753439ae234121a/rmw_implementation_cmake/cmake/call_for_each_rmw_implementation.cmake#L49) itself perhaps this block can be removed again.